README.md

```diff
@@ -1,6 +1,15 @@
 # Northcoders News API
+---
+NOTE TO DEVELOPERS: 
+Please add your own environment variables to any cloned version of this repo in order to successfully connect locally to the two included databses. 
```

Comment:
Just to be really clear, rather than environment variables it is .env files that are needed to be added by anyone downloading your repo. These will be use _in conjunction_ with environment variables to make sure the correct DB is connected to.